CONTRIBUTING.md

```diff
@@ -2,11 +2,30 @@
 
 🎈 We are so happy to have you!
 
-Todo
+We're trying to create a new way of decentralized work!
+
+There are many ways to contribute, including:
+1. Get involved in the protocol design decisions [Joining Discord]() !
+2. [Grab an Issue]()
```

Comment:
Here's the link:

[Grab an Issue](https://github.com/engineer-dao/platform/issues)